cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,41 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
 
+describe('Article Creation and Deletion', () => {
+  let user, article;
+
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.login(user.email, user.username, user.password);
+    });
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
   });
 
-  it('', () => {
+  it('Sign in and Create an Article', () => {
```

Comment:
check the good test names examples over here https://mate-academy.github.io/qa-program/checklists/cypress.html and update